api/lib/domain/services/user-reconciliation-service.js

```diff
@@ -38,12 +53,36 @@ async function findMatchingSchoolingRegistrationIdForGivenOrganizationIdAndUser(
     throw new NotFoundError('There were no schoolingRegistrations matching with names');
   }
 
-  const matchingSchoolingRegistration = _.find(schoolingRegistrations, { 'id': schoolingRegistrationId });
-  if (!_.isNil(matchingSchoolingRegistration.userId)) {
-    throw new SchoolingRegistrationAlreadyLinkedToUserError();
+  const matchedSchoolingRegistration = _.find(schoolingRegistrations, { 'id': schoolingRegistrationId });
+  await checkIfStudentIsAlreadyReconciledOnTheSameOrganization(matchedSchoolingRegistration, userRepository);
+
+  return matchedSchoolingRegistration;
+}
+
+async function checkIfStudentIsAlreadyReconciledOnTheSameOrganization(matchingSchoolingRegistration, userRepository) {
+  if (!_.isNil(matchingSchoolingRegistration.userId))  {
+    const userId = matchingSchoolingRegistration.userId ;
+    const user = await userRepository.getUserAuthenticationMethods(userId);
+    const authentificationMethod = getUserAuthenticationMethodWithObfuscation(user);
+
+    const detail = 'Un compte existe déjà pour l\'élève dans le même établissement.';
+    const error = STUDENT_RECONCILIATION_ERRORS.IN_SAME_ORGANIZATION[authentificationMethod.authenticatedBy];
+    const meta = { displayShortCode: error.displayShortCode, value: authentificationMethod.value };
+    throw new SchoolingRegistrationAlreadyLinkedToUserError(detail, error.code, meta);
   }
+}
 
-  return schoolingRegistrationId;
+async function checkIfStudentHasAlreadyAccountsReconciledInOtherOrganizations(student, userRepository) {
+  if (!_.isEmpty(student) && !_.isEmpty(student.account)) {
```

Comment:
Je crois que ça devrait fonctionné : 
```suggestion
  if (_.get(student, 'account') {
```